src/test/java/tracko/testutil/TypicalOrders.java

```diff
@@ -67,25 +68,39 @@ public class TypicalOrders {
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
-            .withPhone("94351253").withItemList(List.of(DEFAULT_PAIR, PAIR_1)).build();
+            .withPhone("94351253")
+            .withTimeCreated(LocalDateTime.of(2022, 5, 12, 15, 45, 23))
+            .withItemList(List.of(DEFAULT_PAIR, PAIR_1)).build();
     public static final Order ORDER_2 = new OrderBuilder().withName("Benson Meier")
             .withAddress("311, Clementi Ave 2, #02-25").withEmail("johnd@example.com")
-            .withPhone("98765432").withItemList(List.of(DEFAULT_PAIR, PAIR_2)).build();
+            .withPhone("98765432")
+            .withTimeCreated(LocalDateTime.of(2022, 6, 19, 9, 23, 12))
+            .withItemList(List.of(DEFAULT_PAIR, PAIR_2)).build();
     public static final Order ORDER_3 = new OrderBuilder().withName("Carl Kurz")
             .withPhone("95352563").withEmail("heinz@example.com")
-            .withAddress("wall street").withItemList(List.of(DEFAULT_PAIR, PAIR_3)).build();
+            .withAddress("wall street")
+            .withTimeCreated(LocalDateTime.of(2022, 7, 22, 23, 55, 13))
+            .withItemList(List.of(DEFAULT_PAIR, PAIR_3)).build();
     public static final Order ORDER_4 = new OrderBuilder().withName("Daniel Meier")
             .withPhone("87652533").withEmail("cornelia@example.com")
-            .withAddress("10th street").withItemList(List.of(DEFAULT_PAIR, PAIR_4)).build();
+            .withAddress("10th street")
+            .withTimeCreated(LocalDateTime.of(2022, 8, 14, 3, 15, 44))
+            .withItemList(List.of(DEFAULT_PAIR, PAIR_4)).build();
     public static final Order ORDER_5 = new OrderBuilder().withName("Elle Meyer")
             .withPhone("9482224").withEmail("werner@example.com")
-            .withAddress("michegan ave").withItemList(List.of(DEFAULT_PAIR, PAIR_5)).build();
+            .withAddress("michegan ave")
+            .withTimeCreated(LocalDateTime.of(2022, 9, 9, 12, 15, 54))
+            .withItemList(List.of(DEFAULT_PAIR, PAIR_5)).build();
     public static final Order ORDER_6 = new OrderBuilder().withName("Fiona Kunz")
             .withPhone("9482427").withEmail("lydia@example.com")
-            .withAddress("little tokyo").withItemList(List.of(DEFAULT_PAIR, PAIR_6)).build();
+            .withAddress("little tokyo")
+            .withTimeCreated(LocalDateTime.of(2022, 9, 25, 15, 30, 21))
+            .withItemList(List.of(DEFAULT_PAIR, PAIR_6)).build();
     public static final Order ORDER_7 = new OrderBuilder().withName("George Best")
             .withPhone("9482442").withEmail("anna@example.com")
-            .withAddress("4th street").withItemList(List.of(DEFAULT_PAIR, PAIR_7)).build();
+            .withAddress("4th street")
+            .withTimeCreated(LocalDateTime.of(2022, 10, 11, 12, 4, 9))
+            .withItemList(List.of(DEFAULT_PAIR, PAIR_7)).build();
```

Comment:
I also added `timeCreated` to the TypicalOrders in my PR, but I used the `DEFAULT_TIME_CREATED` that's in one of the classes. I think your implementation of the different times is better than using just one default time, but I'm not sure if it's going to affect the tests in EditOrderCommand. Can you help me check (by pulling the latest merge)? So sorry for the troubls :"

Reply:
Oh I see, actually `EditOrderCommand` tests have not been implemented yet, did you mean `EditItemCommand`? 

But it does affect the `SortOrderCommand` tests as the sort doesn't work if all orders have the exact same `timeCreated`. Is it alright if I change the current implementation to use different `timeCreated` for the orders used in `getTypicalOrders()`?

Also, if I'm not wrong, `OrderBuilder` already uses `DEFAULT_TIME_CREATED` when creating orders without specifying a time so there's no need to specify `DEFAULT_TIME_CREATED` actually

Reply:
Oh I see, actually doesn't Java CI already check if the tests work? So if all the checks pass it should be fine if I'm not wrong

Reply:
Ah yep, I'm just slightly unsure, bc previously, the tests wouldn't pass bc timeCreated was not implemented in the OrderBuilder yet. I think can just merge and see; for the most part I think it should work out fine, I'm just not too sure haha

Reply:
Really? If I'm not wrong shouldn't `OrderBuilder` create orders with `DEFAULT_TIME_CREATED` by default?

Reply:
Hmm I personally can't remember what the issue was exactly (with the test cases failing) aside from it being a difference in the `Order` String comparison, but if that's the case then I think the issue was with my `EditOrderCommand` bc it didnt copy over the time to the replacement order.

I think this change shouldn't affect the `EditOrderCommand` test cases then; I'm so sorry for the confusion!

Reply:
No worries, thanks for bringing it up! I'll double check after the merge to see if it's alright!

Reply:
Everything looks alright after the merge!

Reply:
Ok I see the problem now, basically, `Order::equals` doesn't actually check for equality in `timeCreated` but `Order::toString` has a `timeCreated` component to show the user at what time the order was created. However, the test actually checks for equality in the `CommandResult` as well, which uses `Order::toString`, which is why the different `timeCreated` causes different `CommandResults`